docs/source/look_and_feel.rst

```diff
@@ -1,22 +1,93 @@
 Look and Feel
 =============
 
-You can use the workspace "All Text Options" to change the look and feel of document texts.
-You can access these options from the kabab menu on the top right (aka three vertical dots).
-Use this menu to adjust various |app| appearance options including font size and type, text
-and background colors, toggling words of Christ in red, and more.
-
-When using multiple windows, you can use the individual window text options to override
-current workspace settings. Tap and hold the square window icon at the bottom for the
-window whose settings you wish to customize.  Choose "Text Options" then "All Text Options".
-This will open the menu to configure |app|'s' appearance for that specific window.
-
-.. image:: ./images/multi_window_text_options.png
-
-For things that can not be changed through these menus, like the color of certain
-texts within a module, e.g. Strong's numbers, you can code the color in the style.css
-file located within  a zip package. You can use HiSB Bible module to see the structure
-and placement of this file, it should be located in the modules/texts/ztext/hisb/style
+The look and feel of |app| can be configured for the entire application, for
+specific workspaces, specific windows, or even individual modules.
+
+Application
+-----------
+
+To configure application-wide look and feel settings, navigate to the top left
+main menu (|hamburger|) and click `Application preferences`. Some options that
+you can use to customize the look and feel of the application are:
+
+    - **Application language:** Choose the language you would like the application
+      to use in dialogs, settings, etc.
+    - **Black & white mode:** Enabling this setting reduces the use of colors.
+      This is useful for e-ink displays.
+    - **Disable animations:** Enabling this setting will reduce animations that
+      may not work well on older devices.
+    - **Font size multiplier:** Changing this setting will scale document text
+      sizes. It can be useful for devices with larger screens.
```

Comment:
Perhaps good to emphasize that this allows making difference between devices if sync is used. Then user can adjust per-window/per-workspace font setting (which will be synced) and this multiplier is still applied to each device separately.